src/OrchardCore.Modules/OrchardCore.Forms/Views/Items/SelectPart.cshtml

```diff
@@ -0,0 +1,26 @@
+@using OrchardCore.Forms.Models
+@model ShapeViewModel<SelectPart>
+@{
+    var formElementPart = Model.Value.ContentItem.As<FormElementPart>();
+    var formInputElementPart = Model.Value.ContentItem.As<FormInputElementPart>();
+    var elementId = formElementPart.Id;
+    var fieldName = formInputElementPart.Name;
+    var fieldId = !string.IsNullOrEmpty(elementId) ? elementId : !string.IsNullOrEmpty(fieldName) ? Html.GenerateIdFromName(fieldName) : default(string);
```

Comment:
May be default to `contentitemid_nameof(part)` instead of default(string)?

Reply:
again same as the other templates - could be review as a batch

Reply:
ok